PiSendData.py

```diff
@@ -0,0 +1,31 @@
+import socket
+import struct
+import sys
+import imp
+import math
+
+sys.path.append('/usr/local/lib/vmxpi/')
+vmxpi = imp.load_source('vmxpi_hal_python', '/usr/local/lib/vmxpi/vmxpi_hal_python.py')
+
+SERVER_IP = "10.49.99.2"
+SERVER_PORT = 5800
+DELAY = 20 # ms
+
+vmx = vmxpi.VMXPi(False, 50)
+time = vmx.getTime()
+
+out = bytearray(17)
```

Comment:
magic constant.  Must share, or in the case of differing languages, at least break out as a constant once per language.